tests/vm/gh_test.go

```diff
@@ -272,11 +277,13 @@ func TestVmLog(t *testing.T) {
 }
 
 func TestInputLimits(t *testing.T) {
+	t.Skip() // 1413631 lines of test code .. @jeff
```

Comment:
These two run in a couple of seconds in develop, should be OK to leave unskipped?